core/db/entities.py

```diff
@@ -32,6 +32,7 @@ class Submission(BaseModel):
     reporting_period_end = sqla.Column(sqla.DateTime(), nullable=False)
     reporting_round = sqla.Column(sqla.Integer(), nullable=False)
     submission_filename = sqla.Column(sqla.String(), nullable=True)
+    event_data_blob = sqla.Column(JSONB, nullable=True)
```

Comment:
not sure the label of "event data" is strictly speaking accurate here - we should probably just call it `meta_data_blob` or similar?

Reply:
Does the jsonb blob functionality not assume that it will be called `event_data_blob`?